README.rst

```diff
@@ -78,6 +78,7 @@ Example:
         "IBAN": "NL50BANK1234567890",
         "BIC": "BANKNL2A",
         "batch": True,
+        # For non-SEPA transfers, set "domestic", necessary e.g. for CH/LI
```

Comment:
It feels non-intuitive to check for the presence of a config option, could we say ``"domestic": True`` is required for domestic transfers and ``"domestic": False`` as the default?

Reply:
Good idea, changed documentation and code accordingly; see 705d1a00